charts/operator/rbac.go

```diff
@@ -220,6 +225,21 @@ func ClusterRole(dot *helmette.Dot) []rbacv1.ClusterRole {
 						APIGroups: []string{"cluster.redpanda.com"},
 						Resources: []string{"topics/status"},
 					},
+					{
+						Verbs:     []string{"get", "list", "patch", "update", "watch"},
+						APIGroups: []string{"cluster.redpanda.com"},
+						Resources: []string{"users"},
```

Comment:
oh gosh, I just now saw the differentiation here between
```go
values.Scope == Namespace
```
and 
```go
values.Scope == Cluster
```

if you're adding this here (under `Cluster`) we'll likely want to do `schemas` as well. Then again is the `Cluster` scope intended to be for the `v1` operator based on the [bootup logic for the operator](https://github.com/redpanda-data/redpanda-operator/blob/3a34a83aa5ac5c2cdc6b77cf125575873a047980/operator/cmd/run/run.go#L233-L236)?

Either way I think that we'll want to either drop this (if it is solely for the v1 controller) or add in `schemas` as well.

Reply:
Fixed. To be on the safe side, I'm leaving the V2 permissions in V1 as they won't have an adverse affect if they're not installed or used AFAIK. Ideally, we'll eventually remove all the gating.